notebooker/web/templates/results.html

```diff
@@ -21,6 +21,7 @@ <h2>{{ result.report_title }}</h2>
                         class="iframeToLoad"
                         scrolling="no">
                 </iframe>
+                <code id="stdoutBlock"></code>
```

Comment:
Maybe we can make this copy-pasteable by putting it in a textarea? Ideally either in a modal/popup or a new plaintext tab